core/src/main/java/org/apache/accumulo/core/clientImpl/InstanceOperationsImpl.java

```diff
@@ -274,6 +287,43 @@ public List<ActiveScan> getActiveScans(String tserver)
     }
   }
 
+  @Override
+  public List<ActiveScan> getActiveScans(ServerId server)
+      throws AccumuloException, AccumuloSecurityException {
+
+    @SuppressWarnings("unused")
+    var unused = Objects.nonNull(server);
+    Preconditions.checkArgument(
+        server.getType() == ServerTypeName.SCAN_SERVER
+            || server.getType() == ServerTypeName.TABLET_SERVER,
+        "Server type %s is not %s or %s.", server.getType(), ServerTypeName.SCAN_SERVER,
+        ServerTypeName.TABLET_SERVER);
+
+    final var parsedTserver = HostAndPort.fromParts(server.getHost(), server.getPort());
+    TabletScanClientService.Client client = null;
+    try {
+      client = getClient(ThriftClientTypes.TABLET_SCAN, parsedTserver, context);
+
+      List<ActiveScan> as = new ArrayList<>();
+      for (var activeScan : client.getActiveScans(TraceUtil.traceInfo(), context.rpcCreds())) {
+        try {
+          as.add(new ActiveScanImpl(context, activeScan));
+        } catch (TableNotFoundException e) {
+          throw new AccumuloException(e);
+        }
+      }
+      return as;
+    } catch (ThriftSecurityException e) {
+      throw new AccumuloSecurityException(e.user, e.code, e);
+    } catch (TException e) {
+      throw new AccumuloException(e);
+    } finally {
+      if (client != null) {
+        returnClient(client, context);
```

Comment:
It's a little confusing to have the thrift client named client, and the AccumuloClient to be called context in the AccumuloClient-adjacent code. It might be less confusing if client was called something like `rpcClient` or similar. That said, it doesn't really matter... it just caused a brief double-take, which is not really a very strong justification for changing things.

Reply:
Changed to `rpcClient` in de7df88